Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -104,7 +95,7 @@ public BlockResult ExecuteNextBlock(CancellationToken cancellationToken)
                     }
                 }
 
-                this.AcceptBlock(new ContextInformation(result, this.Validator.ConsensusParams));
+                this.AcceptBlock(new ContextInformation(result, this.Tip, this.Validator.ConsensusParams));
```

Comment:
I don't think you need to pass in the tip here, tip should only ever be set from this thread right?

Reply:
you may be right, i've added this while doing 413, but maybe it is no longer needed, i'll check that

Reply:
yes, i think you are right, but only after PR #549, which solves the issue completely, so i'll remove this as a part of PR #549

Reply:
fixed there